clothing_rental/accounts/templates/register.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+
+{% block title %}Sign up{% endblock title %}
+
+{% block body %}
+    <br><br>
+
+    {% if CustomUser %}
+    <div>Welcome {{CustomUser.first_name}}! Your account for {{ CustomUser.email }}  has been successfully created!</div>
+    {% endif %}
```

Comment:
To się kiedykolwiek wyświetla?

Reply:
nie, narazie się nie wyświetla i muszę to poprawić, narazie usuwał z aktualnego pulla